addon/templates/components/polaris-resource-list/item.hbs

```diff
@@ -88,7 +92,9 @@
               destructive=shortcutAction.destructive
               icon=shortcutAction.icon
               loading=shortcutAction.loading
-              onClick=(action shortcutAction.onAction)
+              onClick=(action
+                (or shortcutAction.onAction (action (mut dummy)))
```

Comment:
@andrewpye why would there be a shortcutAction button without an action?

Reply:
Could be a `url` action for a link 😉 (I hit this testing the examples in the dummy app)

Reply:
Ohh right.